source/_posts/How-to-add-Lottie-Animation-in-an-Android-app.md

```diff
@@ -0,0 +1,150 @@
+---
+title: How to add Lottie Animation in an Android application
+date: 2020-08-26 18:40:48
+author: Diwakar Singh
+category: Android
+tags:
```

Comment:
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation

Need to put this tag in the blog and in the content